### DISCUSSION TITLE
Please add option to On& Off the wide hit box on mobile in Mania Mode

### DISCUSSION THREAD

**Discussion by gaikoz11** [ppy/osu#27619] [Ideas]
Its hard to play with keyboard on mania in mobile because its too wide  .. but don't get me wrong i love the wide hit box for touch screen for when i try to play with keyboard connected its hard to look on the notes . Also for 7 keys and up .. 

Btw its not only me its also for other mobile platform user i check in YouTube when i upload the video they say the same things

**Comment by Rekunan** [comment]
There was a proposal to improve this, would this satisfy ur needs? #27430
